Avatar.html

```diff
@@ -11,13 +11,48 @@
     <div class="container">
       <h1>Avatar</h1>
     </div>
-    <hr>
+    <hr />
+    <h3>Avatar Demo</h3>
+    <p>
+      Use the Avatar Component as a graphical representation of a person through a profile picture or image, an icon, or a string with initials
+    </p>
     <main class="main">
-      <img class="avatar" src="http://placeimg.com/80/80/people" />
```

Comment:
You missed alt attribute. It will help in increasing your accessibility.

Reply:
Ok thanks Sainath will Implement all points